### PR TITLE
docs: blog about fundraising and v0.3.0 release

### PR DESCRIPTION
**Pull request by gyxlucy**
1. add blog
2. add a "release" tag

### CODE REVIEW

File: website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+authors: [ meng, gyxlucy ]
+
+tags: [release]
+
+---
+# Announcing our $3.2M seed round, and the long-awaited RAG release in Tabby v0.3.0
+
+We are excited to announce that TabbyML has raised a [$3.2M seed round](https://techcrunch.com/2023/10/10/tabbyml-github-copilot-alternative-raises-3-2-million/) to move towards our goal of building an open ecosystem to supercharge developer experience with LLM 🎉🎉🎉. 
+
+## Why TabbyML?
```
Comment (wsxiaoys) [inline website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md:11]: Tabby?

File: website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+With over 18 years coding experience, we recognize the transformative potential of LLMs in reshaping developer toolchains. While many existing products lean heavily on cloud-based end-to-end solutions, we firmly believe that for AI to be genuinely the core of every developer's toolkit, the next-gen LLM-enhanced developer tools should embrace an open ecosystem. This approach promotes not just flexibility for easy customization, but also fortifies security.
```
Comment (wsxiaoys) [inline website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md:12]:
```suggestion
With over 10 years coding experience, we recognize the transformative potential of LLMs in reshaping developer toolchains. While many existing products lean heavily on cloud-based end-to-end solutions, we firmly believe that for AI to be genuinely the core of every developer's toolkit, the next-gen LLM-enhanced developer tools should embrace an open ecosystem. This approach promotes not just flexibility for easy customization, but also fortifies security.
```

18 sounds too bragging

File: website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md
Comment (wsxiaoys) [inline website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md:1]: maybe adjust date to make it list earlier than the rag one？

**Review by wsxiaoys — CHANGES_REQUESTED** [review]

File: website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+authors: [ meng, gyxlucy ]
+
+tags: [release]
+
+---
+# Announcing our $3.2M seed round, and the long-awaited RAG release in Tabby v0.3.0
```
Comment (wsxiaoys) [inline website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md:7]: consider this is the blog for `tabby`, let move this announcement to the bottom of blog post?

Comment (gyxlucy) [reply website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md:7]: Actually this blog is more about fundraising so let's keep it this way.

**Pull request merged**